hardware/tb/verilator/verilator.flags

```diff
@@ -9,6 +9,7 @@
 -Wno-BLKANDNBLK
 -Wno-CASEINCOMPLETE
 -Wno-CMPCONST
+-Wno-EOFNEWLINE
```

Comment:
Why do we need this now? Could we just fix the warnings?

Reply:
Done